src/pypowsybl_jupyter/selectcontext.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2024, RTE (http://www.rte-france.com)
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+#
+
+from collections import OrderedDict
+import pandas as pd
+from pypowsybl.network import Network
+
+class SelectContext:
+
+    def __init__(self, network:Network = None, vl_id : str = None, use_name:bool = True):
+        self.network = network
+        self.use_name = use_name
+
+        self.vls = network.get_voltage_levels(attributes=['name'])
+        self.vls['id'] = self.vls.index
+        if use_name:
+            self.vls['name'] = self.vls['name'].replace('', pd.NA).fillna(self.vls['id'])
+        else:
+            self.vls['name'] = self.vls.index
```

Comment:
A bit strange, aren't you filling here the "name" column with the ids? I'd rather expect a column variable which would either be "name" or "id".